### PR TITLE
Production Release as on 12-02-2020

### PR DESCRIPTION
**Pull request by rabbi08**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_

- [ ] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [ ] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [ ] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/components/Feed/PostCard.js [hunk 1]
```diff
@@ -13,25 +13,50 @@ const PostCard = styled(Card)`
   margin-bottom: 4rem;
   padding: 2rem 2rem 0rem 2rem;
   overflow-wrap: break-word;
+  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
```
Comment (idjevm) [inline client/src/components/Feed/PostCard.js:16]: Not a blocking change but this shouuld be updated to rems later on...

File: client/src/components/Feed/PostDeleteModal.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        border-radius: 10px;
+        .ant-modal-header {
+            border-bottom: 1px solid #C5C5C5;
+            border-radius: 10px 10px 0 0;
+            padding-bottom: 0;
+            .ant-modal-title {
+                text-align: center;
+                font-weight: 700;
+                font-size: ${large};
+            }
```
Comment (idjevm) [inline client/src/components/Feed/PostDeleteModal.js]: Same idea as above in this file, there is a mix here, rems & pxs, just use `rem`... @Zaydme or anyone else, keep a note to update later on...

**Review by idjevm — APPROVED** [review]
Codewise looks good, nothing major... .

**Comment by adorapham** [comment]
I have verified these PRs -
Fix delete post/comment modal UI #1932;   fix fonts for Vietnamese, (and other languages)#1934 ;  #1394 - added all special Characters used by auth0 to F/E password ;  #1948 - fix "view help board" button visibility #1956 ;  #1916 - added missing translation keys for messaging and notifications #1959 ; #1940 - sort languages alphabetically (#1965) ; #1893 - Feed Message Modal Initialize On Cancel (#1938) ; #1931 - Modify labels on post cards #1976  ;#1975 - Add new types of Help #1977 ;Lokalise: Translations update #1996.
productVerify1 (1)

#1947 - shift enter for new line when creating comment (#1958); 
N/A - Change spanish display name (#1970); "The Post is displayed as 'edited' after user 'Like' the post" #1952; verified
image

**Comment by jessica-lau** [comment]
Retested:
#1957, #1965, #1958, #1983, #1969, #2013 and automated tests #1937 #1941 #1864

**Comment by galeza** [comment]
Retested: #1992 , #1990 , #1996

**Review by jessica-lau — APPROVED** [review]

**Review by Zaydme — APPROVED** [review]

**Review by galeza — APPROVED** [review]
LGTM

**Review by FedericaZn — APPROVED** [review]

**Pull request merged**